sdk/src/Services/DynamoDBv2/Custom/DataModel/InternalModel.cs

```diff
@@ -50,22 +48,25 @@ internal class SimplePropertyStorage
         public MemberInfo Member { get; protected set; }
         // Type of the property
 
-#if NET8_0_OR_GREATER
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)]
```

Comment:
Is changing from `PublicParameterlessConstructor` to `DataModelModeledType` here intentional?

Reply:
I can't remember it was set to `PublicParameterlessConstructor` before but since the `MemberType` can represent a type from the user and this method relies on methods that also use the `DataModelModeledType` I need to make it consistent here.